views/fe-utils/tooltipUtil.js

```diff
@@ -1,2 +1,7 @@
-export const toolTipPercent = (count, targetTotal) =>
-  ((+count / +targetTotal) * 100).toFixed(0)
+export const toolTipPercent = (count, targetTotal) => {
+  if (Number.isNaN(+count) || Number.isNaN(+targetTotal)) {
+    return 0
+  }
+
+  return ((+count / +targetTotal) * 100).toFixed(0)
+}
```

Comment:
Same note here on possibly dividing by `0`
```suggestion
export const toolTipPercent = (count, targetTotal) => {
  if (!targetTotal || Number.isNaN(+count) || Number.isNaN(+targetTotal)) {
    return 0
  }

  return ((+count / +targetTotal) * 100).toFixed(0)
}
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/159/commits/bac9a58114af036773a1082131399f1c12425c2d